js/ui/interaction.js

```diff
@@ -49,12 +49,12 @@ function Interaction(el) {
         now = null;
     }
 
-    function click(point) {
-        interaction.fire('click', {point: point});
+    function click(point, ev) {
+        interaction.fire('click', {point: point, raw: ev});
```

Comment:
Can we rename this to `originalEvent` to match jQuery and make it a little bit more obvious? `raw` is very terse and ambiguous